### ISSUE TITLE
Open log view in new tab

### ISSUE THREAD

**Issue by michellutz** [etf-validator/governance#34]
# Background and Motivation
The log view does not have any way to return to the etf-webapp.

# Proposed change
Add the etf navigation bar to the log view.

# Alternatives
Add the ETF header to the html log page.

# Funding
Ensured by JRC

**Comment by jenriquesoriano** [comment]
It is proposed to open the log view in a new tab

**Comment by carlospzurita** [comment]
This improvement has been developed and can be found on the Guadaltel fork, in the branch enhancement-sprint-october. Link to the commit https://github.com/guadaltel/etf-webapp/commit/c243645b47d00206cd5a8a2f24b65efeb9ef56ea

**Comment by michellutz** [comment]
SG agrees to this being moved into release planning, unless there are concerns by the TC.

**Comment by carlospzurita** [comment]
In order to integrate the changes on https://github.com/guadaltel/etf-webapp/commit/c243645b47d00206cd5a8a2f24b65efeb9ef56ea , it would be advisable to solve the pull request https://github.com/etf-validator/etf-webapp/pull/158 . This way, a new pull request can be safely created, without risk of muddling up the previous one, and the merge can be solved far easier.

This also applies to the issues #33 #35 and #48 that are still pending on review and integrate the changes. This approach is much more agile, and let the reviewer check all the changes at glance.

**Comment by jonherrmann** [comment]
Implemented in [Version 2.1.0](https://github.com/etf-validator/etf-webapp/releases/tag/2.1.0)
